cli_reference/osdk/cli-osdk-install.adoc

```diff
@@ -11,7 +11,7 @@ See xref:../../operators/operator_sdk/osdk-about.adoc#osdk-about[Developing Oper
 
 [NOTE]
 ====
-{product-title} 4.8 and later supports Operator SDK v1.8.0.
+{product-title} 4.9 and later supports Operator SDK v1.10.0.
```

Comment:
operator-sdk version is "v1.10.1-ocp" for OCP4.9

Reply:
@emmajiafan Thanks, I've updated the version to just be "v1.10.1" when it's mentioned in sentences. For the [`operator-sdk version` command example output](https://deploy-preview-36521--osdocs.netlify.app/openshift-enterprise/latest/operators/operator_sdk/osdk-installing-cli.html#osdk-installing-cli-linux-macos_osdk-installing-cli), however, it will show more specifically as "v1.10.1-ocp".